airflow-core/src/airflow/ui/src/i18n/locales/he/common.json

```
@@ -0,0 +1,79 @@
+{
+  "defaultToGraphView": "תצוגת גרף כברירת מחדל",
+  "defaultToGridView": "תצוגת רשת כברירת מחדל",
+  "logout": "התנתק",
+  "switchToDarkMode": "החלף למצב כהה",
+  "switchToLightMode": "החלף למצב בהיר",
+  "timezone": "איזור זמן",
+  "user": "משתמש",
+  "language": {
+    "chinese": "סינית מסורתית",
+    "english": "אנגלית",
+    "german": "גרמנית",
+    "korean": "קוריאנית",
+    "select": "בחר"
+  },
+  "nav": {
+    "home": "מסך הבית",
+    "assets": "נכסים (Assets)",
+    "browse": "עיין",
+    "admin": "מנהל",
+    "docs": "תיעוד",
+    "plugins": "פלאגינים"
+  },
+  "browse": {
+    "auditLog": "לוג מערכת",
+    "xcoms": "(XComs)"
+  },
+  "admin": {
+    "Variables": "משתנים",
+    "Pools": "בריכת משאבים",
+    "Providers": "חבילות עזר",
+    "Plugins": "פלאגינים",
+    "Connections": "חיבורים",
+    "Config": "קונפיגורציות"
+  },
+  "timeRange": {
+    "duration": "משך זמן",
+    "lastHour": "השעה האחרונה",
+    "last12Hours": "12 השעות האחרונות",
+    "last24Hours": "24 השעות האחרונות",
+    "pastWeek": "השבוע האחרון"
+  },
+  "docs": {
+    "documentation": "דוקומנטציה",
+    "githubRepo": "GitHub קוד ב",
```

`קוד` is quite misleading - it should be a repository
```suggestion
    "githubRepo": "GitHub ריפו ב",
```

or 

```suggestion
    "githubRepo": "GitHub ריפוזיטורי ב",
```

Or leave it as-is in English.